command-expansion-server/jestGlobalSetup.js

```diff
@@ -0,0 +1,68 @@
+import { config } from 'dotenv';
+config();
+import fs from 'fs';
+import os from 'os';
+import path from 'path';
+import fetch from 'node-fetch';
+import { Writable } from 'stream';
+import readline from 'node:readline/promises';
```

Comment:
Note: This import breaks on node 16.4.0, but works on 17.8.0. [This line](https://github.com/NASA-AMMOS/aerie/blob/78cb0e4efe12f9e4f989087bbc6030498a606a61/command-expansion-server/build.gradle#L6) seems to imply that we're using `16.4.0`.

Although...even after installing node 17 I'm still unable to get the tests to pass. Maybe one of you could give me a quick tutorial.

Reply:
Testing node version requirement is higher than deployment version, yes.